https://issues.redhat.com/browse/ACM-6365 (#5294)
open-cluster-management/rhacm-docs: release_notes/known_issues_console.adoc

== _EditApplicationSet_ expand feature repeats

When you add multiple label expressions or attempt to enter your cluster selector for your `ApplicationSet`, you might receive the following message repeatedly,  "Expand to enter expression". You can enter your cluster selection despite this issue.

[#No-longer-supporting-Argo-CD-pull-model]
== Application console does not support Argo CD pull model
// 2.8.6365

> When you deploy `ApplicationSet` applications using the Argo CD pull model, the application will not display correctly from the _Topology_ page as they are not supported. 

Reviewer: ```suggestion

When you deploy `ApplicationSet` applications using the Argo CD pull model, the application is displayed incorrectly from the _Topology_ page. Deploying `ApplicationSet` applications using the Argo CD pull model is not supported. 
```

> // 2.8.6365
> When you deploy `ApplicationSet` applications using the Argo CD pull model, the application will not display correctly from the _Topology_ page as they are no longer supported. 

Reviewer: ```suggestion
// 2.8.6365

When you deploy `ApplicationSet` applications using the Argo CD pull model, the application is displayed incorrectly from the _Topology_ page.
```

@fxiang1 should we also remove the following statement: "Click the Topology tab for visual representation of all the applications and resources in your project." from [Application console](https://github.com/stolostron/rhacm-docs/blob/2.8_stage/applications/app_console.adoc).

Reviewer 2: @dockerymick Nope, we don't need to change anything else in the docs as they should still be working for the ArgoCD push model and other application types.

Reviewer: @fxiang1 thanks for confirming!

> When you deploy `ApplicationSet` applications using the Argo CD pull model, the application will not display correctly from the _Topology_ page as they are no longer supported. 

Reviewer 2: Thanks @faaden123, I think this looks really good now.

I just want to make one minor change which is to replace:
`no longer`
with
`not`

Reason being we never had this support before and we are planning to support this in future releases. Thanks!

Author: @fxiang1 Gotcha- fixed. Thank you for the clarification!

== _EditApplicationSet_ expand feature repeats

When you add multiple label expressions or attempt to enter your cluster selector for your `ApplicationSet`, you might receive the following message repeatedly,  "Expand to enter expression". You can enter your cluster selection despite this issue.

[#No-longer-supporting-Argo-CD-pull-model]
== Application console does not support Argo CD pull model
// 2.8.6365

> When you deploy `ApplicationSet` applications using the Argo CD pull model, the application is displayed incorrectly from the _Topology_ page. Deploying `ApplicationSet` applications using the Argo CD pull model is not supported. 

Reviewer: One more update, otherwise this looks good

```suggestion
When you deploy `ApplicationSet` resources using the Argo CD pull model, the application is displayed incorrectly from the _Topology_ page. Deploying `ApplicationSet` applications using the Argo CD pull model is not supported. 
```
